Cargo.toml

```diff
@@ -7,17 +7,15 @@ authors = ["Mcat12 <newtoncat12@yahoo.com>", "0xcaff <martincharles07@gmail.com>
 lto = true
 
 [dependencies]
+diesel = { version = "1.0.0", features = ["postgres", "sqlite"] }
```

Comment:
Are we supporting both postgres and sqlite? The code only supports SQLite ATM.

Reply:
Good catch. I used SQLite because it's easy for testing. Hmm. Need to think about this a bit more. There's a generic `Connection` exposed by diesel which could be used instead.